velox/functions/sparksql/fuzzer/tests/SparkQueryRunnerTest.cpp

```diff
@@ -26,6 +26,7 @@
 #include "velox/parse/TypeResolver.h"
 #include "velox/vector/tests/utils/VectorTestBase.h"
 
+using namespace facebook;
```

Comment:
Why we need additional facebook namespace?

Reply:
For making complier not confused with the newly added test namespace in exec.